crates/eframe/src/native/glow_integration.rs

```diff
@@ -552,44 +552,6 @@ impl GlowWinitRunning {
             (raw_input, viewport_ui_cb)
         };
 
-        let clear_color = self
-            .app
-            .clear_color(&self.integration.egui_ctx.style().visuals);
-
-        let has_many_viewports = self.glutin.borrow().viewports.len() > 1;
-        let clear_before_update = !has_many_viewports; // HACK: for some reason, an early clear doesn't "take" on Mac with multiple viewports.
-
-        if clear_before_update {
-            // clear before we call update, so users can paint between clear-color and egui windows:
-
-            let mut glutin = self.glutin.borrow_mut();
-            let GlutinWindowContext {
-                viewports,
-                current_gl_context,
-                not_current_gl_context,
-                ..
-            } = &mut *glutin;
-            let viewport = &viewports[&viewport_id];
-            let Some(window) = viewport.window.as_ref() else {
-                return Ok(EventResult::Wait);
-            };
-            let Some(gl_surface) = viewport.gl_surface.as_ref() else {
-                return Ok(EventResult::Wait);
-            };
-
-            let screen_size_in_pixels: [u32; 2] = window.inner_size().into();
-
-            {
-                frame_timer.pause();
-                change_gl_context(current_gl_context, not_current_gl_context, gl_surface);
-                frame_timer.resume();
-            }
-
-            self.painter
-                .borrow()
-                .clear(screen_size_in_pixels, clear_color);
-        }
```

Comment:
Doesn't this break `test_inline_glow_paint`?

Reply:
> Doesn't this break `test_inline_glow_paint`?

Yes, no problem. 
(The plan was to PR wgpu and glow separately.)

Reply:
> Doesn't this break `test_inline_glow_paint`?

As planned, I will test glow separately and if there are no issues, I will PR again.